smoke-tests/apps/SpringBoot/src/smokeTest/java/com/microsoft/applicationinsights/smoketest/SpringBootTest.java

```diff
@@ -60,23 +58,8 @@ void testResultCodeWhenRestControllerThrows() throws Exception {
     List<Envelope> rdList = testing.mockedIngestion.waitForItems("RequestData", 1);
 
     Envelope rdEnvelope = rdList.get(0);
-    String operationId = rdEnvelope.getTags().get("ai.operation.id");
-    List<Envelope> edList =
-        testing.mockedIngestion.waitForItems(
-            input -> {
-              if (!"ExceptionData".equals(input.getData().getBaseType())) {
-                return false;
-              }
-              if (!operationId.equals(input.getTags().get("ai.operation.id"))) {
-                return false;
-              }
-              // lastly, filter out ExceptionData captured from tomcat logger
-              ExceptionData data = (ExceptionData) ((Data<?>) input.getData()).getBaseData();
-              return !data.getProperties().containsKey("LoggerName");
-            },
-            1,
-            10,
-            TimeUnit.SECONDS);
+    List<Envelope> edList = testing.mockedIngestion.waitForItems("ExceptionData", 1);
+    assertThat(edList.size()).isEqualTo(2);
```

Comment:
why list size 2 here?

Reply:
Could you please add a code comment about why 2